packages/protocol/contracts/stability/SortedOracles.sol

```diff
@@ -2,23 +2,25 @@ pragma solidity ^0.5.8;
 
 import "openzeppelin-solidity/contracts/math/SafeMath.sol";
 import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
-import "./SortedFractionMedianList.sol";
 import "./interfaces/ISortedOracles.sol";
 import "../common/Initializable.sol";
+import "../common/linkedlists/AddressSortedLinkedListWithMedian.sol";
+import "../common/linkedlists/SortedLinkedListWithMedian.sol";
 
 
 /**
  * @title Maintains a sorted list of oracle exchange rates between Celo Gold and other currencies.
  */
 contract SortedOracles is ISortedOracles, Ownable, Initializable {
   using SafeMath for uint256;
-  using SafeMath for uint128;
-  using SortedFractionMedianList for SortedFractionMedianList.List;
+  using AddressSortedLinkedListWithMedian for SortedLinkedListWithMedian.List;
+  // All oracle rates are assumed to have a denominator of 2 ^ 64.
+  uint256 public constant DENOMINATOR = 0x10000000000000000;
```

Comment:
What do you think about waiting for Fixidity to be merged and use it here, to not add another ad-hoc fixed fraction implementation?

Reply:
Will fixidity work out of the box with this? Would rather get this in sooner for audit purposes, happy to move to fixidity in a follow-up PR

Reply:
It should. Comparisons on Fixidity fractions are the same as on integers.

Reply:
Values are uint256's for all of these implementations